core/statix.multilang.eclipse/src/main/java/mb/statix/multilang/eclipse/AnalysisContextInitializer.java

```diff
@@ -0,0 +1,52 @@
+package mb.statix.multilang.eclipse;
+
+import mb.log.api.Logger;
+import mb.spoofax.eclipse.SpoofaxPlugin;
+import mb.statix.multilang.AnalysisContextService;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IConfigurationElement;
+import org.eclipse.core.runtime.IExtensionRegistry;
+
+import java.util.stream.Stream;
+
+public class AnalysisContextInitializer {
+    private static final String ANALYSIS_CONTEXT_ID = "mb.metaborg.statix.multilang.analysiscontext";
```

Comment:
`mb` == `metaborg`, so `mb.metaborg` is double. Also, we only use `mb` in packages, not in other identifiers, so `mb.metaborg` needs to be removed. There's multiple places where this happens.